src/lib/cozy-client/DataAccessFacade.js

```diff
@@ -18,7 +22,13 @@ export default class DataAccessFacade {
   }
 
   setup(cozyUrl, options) {
-    const config = { cozyURL: cozyUrl, ...options }
+    // prepend the protocol if there isn't one, required for pouchdb
+    if (/^http(s):/.test(cozyUrl) === false) {
```

Comment:
`test` is already returning a boolean, so I guess we should write
```js
if (!/^http(s):/.test(cozyUrl) {
```

Reply:
What is the difference between `/^http(s):/` and `/^https:/`?

Reply:
You're right about the `false` comparison :). `/^http(s):/` allows for both `http` and `https` protocol.